### ISSUE TITLE
Permission Loophole

### ISSUE THREAD

**Issue by KateWasHere001** [rev1si0n/lamda#95]
- Device: [  Pixel6 ]
- OS version: [ 12 ]
- env: [ kernelsu ]
- installed module: [ Zygisk - LSPosed V1.9.2 , Zygisk Next v1.1.0 ]

"Native test" app v27 will tell "Permission Loophole" after enable this moudle.

**Comment by rev1si0n** [comment]
yes, lamda will relax some selinux domains, which may cause the above problems. do you have more details about this report? do you have the name of the app that reported this error?

**Comment by KateWasHere001** [comment]
"NativeTest" [app](https://yun.daxiaamu.com/files/%E5%B8%B8%E7%94%A8APP/%E6%A3%80%E6%B5%8BROOT%E7%8E%AF%E5%A2%83%E7%9A%84%E8%BD%AF%E4%BB%B6/NativeTest/NativeTest-v27-e698e2e-release.apk?preview):

you may have a test on your device.

**Comment by rev1si0n** [comment]
this problem has been solved and will be released in the next version.

**Comment by rev1si0n** [comment]
已发布

**Comment by KateWasHere001** [comment]
Great!

**Comment by KateWasHere001** [comment]
Sorry to reopen this issue, found another situation.

Device: [ Pixel6 ]
OS version: [ 12 ]
env: [ kernelsu ]
installed module: [ Zygisk - LSPosed V1.9.2 , Zygisk Next v1.1.0 ]
lamda version: 7.71

"Native test" app still tell Permission Loophole and Abnormal Environment.

**Comment by rev1si0n** [comment]
this change is tested on android 10.0 but 12.0 is not, i'll working on it.

**Comment by KateWasHere001** [comment]
Hi, any update ?

**Comment by KateWasHere001** [comment]
> Abnormal Environment
> Property Modified

Is there any change to fix these two ?
